### PR TITLE
Update to 6.6.1

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin please rerender

### CODE REVIEW

**Comment by hmaarrfk** [comment]
we may need to rebuild it with openssl 3.1 instead of 3.2

**Comment by jschueller** [comment]
thanks a lot

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-{% set version = "6.6.0" %}
+{% set version = "6.6.1" %}
 package:
   name: qt6-wayland
   version: {{ version }}
 
 source:
   url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/submodules/qtwayland-everywhere-src-{{ version }}.tar.xz
-  sha256: 30c3d82d85d050c83370710a26d12c802199c51eba5c2726e9d6243edd03b7e8
+  md5: 9380ac5d5ad0b83022db35f3b7040a9e
```
Comment (matthiasdiener) [inline recipe/meta.yaml:8]: Why the change to md5?

Comment (jschueller) [reply recipe/meta.yaml:8]: the qt repo has a md5sums.txt; its easier to read that than to do it manually:
https://mirrors.20i.com/pub/qt.io/archive/qt/6.6/6.6.1/submodules/md5sums.txt

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
Thanks!!!!

are you using Wayland now?

**Comment by jschueller** [comment]
no, not from conda
